src/Ansatz/MPS.jl

```diff
@@ -62,6 +62,30 @@ function MPS(arrays::Vector{<:AbstractArray}; order=defaultorder(MPS))
     return MPS(ansatz, NonCanonical())
 end
 
+MPS(arraysdims::NTuple{N,<:Vector{Int}}; order=defaultorder(MPS)) where {N} = MPS(collect(arraysdims); order=order)
+MPS(arraysdims::Vector{<:Tuple{Vararg{Int}}}; order=defaultorder(MPS)) = MPS(collect.(arraysdims); order=order)
+function MPS(arraysdims::NTuple{N,Tuple{Vararg{Int}}}; order=defaultorder(MPS)) where {N}
+    return MPS(collect(collect.(arraysdims)); order=order)
+end
+function MPS(arraysdims::Vector{<:Vector{Int}}; order=defaultorder(MPS))
```

Comment:
I would use `Base.identity`, not a constructor since you're not passing arrays.
```suggestion
function Base.identity(::Type{MPS}, arraysdims; order=defaultorder(MPS))
```

Reply:
This was one possibility along with the one @jofrevalles suggested below of using `ones`. I replaced the constructor by `identity` as I think it fits better than `ones`.